### ISSUE TITLE
lint-errors

### ISSUE THREAD

**Issue by themr0c** [ansible/molecule#1694]
# Issue Type

- Bug report

Detail any linters or test runners used:

# Desired Behavior

Hally linter :)

# Actual Behaviour

```
flake8 
./molecule/interpolation.py:53: [E501] line too long (80 > 79 characters)
./molecule/dependency/shell.py:37: [E501] line too long (81 > 79 characters)
./molecule/provisioner/ansible_playbook.py:53: [E501] line too long (81 > 79 characters)
./molecule/verifier/goss.py:42: [E501] line too long (81 > 79 characters)
./molecule/verifier/testinfra.py:43: [E501] line too long (82 > 79 characters)
```

**Comment by themr0c** [comment]
additionally, there are more errors coming with flake8==3.6.0 ...

```
time tox -e py37-lint
py37-lint recreate: /home/.ecryptfs/src/externals/themr0c/molecule/.tox/py37-lint
py37-lint installdeps: flake8==3.6.0, yamllint
py37-lint installed: flake8==3.6.0,mccabe==0.6.1,molecule==2.19.1.dev98,pathspec==0.5.9,pycodestyle==2.4.0,pyflakes==2.0.0,PyYAML==3.13,yamllint==1.14.0
py37-lint run-test-pre: PYTHONHASHSEED='297123935'
py37-lint runtests: commands[0] | flake8
./molecule/shell.py:57: [W504] line break after binary operator
./molecule/shell.py:67: [W504] line break after binary operator
./molecule/driver/base.py:139: [W504] line break after binary operator
./test/functional/docker/test_scenarios.py:352: [W605] invalid escape sequence '\['
./test/functional/docker/test_scenarios.py:352: [W605] invalid escape sequence '\]'
./test/functional/docker/test_scenarios.py:352: [W605] invalid escape sequence '\['
./test/functional/docker/test_scenarios.py:352: [W605] invalid escape sequence '\]'
./test/functional/docker/test_scenarios.py:353: [W605] invalid escape sequence '\['
./test/functional/docker/test_scenarios.py:353: [W605] invalid escape sequence '\]'
./test/functional/docker/test_scenarios.py:353: [W605] invalid escape sequence '\['
./test/functional/docker/test_scenarios.py:353: [W605] invalid escape sequence '\]'
./test/functional/docker/test_scenarios.py:354: [W605] invalid escape sequence '\['
./test/functional/docker/test_scenarios.py:354: [W605] invalid escape sequence '\]'
./test/functional/docker/test_scenarios.py:354: [W605] invalid escape sequence '\['
./test/functional/docker/test_scenarios.py:354: [W605] invalid escape sequence '\]'
ERROR: InvocationError for command '/home/.ecryptfs/src/externals/themr0c/molecule/.tox/py37-lint/bin/flake8' (exited with code 1)
_______________________________________________ summary ________________________________________________
ERROR:   py37-lint: commands failed

real	0m12.108s
user	0m9.563s
sys	0m1.338s
```

**Comment by themr0c** [comment]
I'll pin the version of flake8 to 3.5.0 because I am not sure how to fiw the issues brought by flake8==3.6.0

**Comment by themr0c** [comment]
OK I will add the fixes for flake8>=3.6.0,4
I will also make sure the versions of the packages are defined only in one single place.
